x/validators/msg.go

```diff
@@ -21,10 +20,10 @@ func (*SetValidatorsMsg) Path() string {
 func (m ValidatorUpdate) Validate() error {
 	if len(m.Pubkey.Data) != 32 ||
 		strings.ToLower(m.Pubkey.Type) != "ed25519" {
-		return errors.WithCode(errInvalidPubKey, CodeInvalidPubKey)
+		return InvalidPubKeyErr
 	}
 	if m.Power < 0 {
-		return errors.WithCode(errInvalidPower, CodeInvalidPower)
+		return InvalidPower
```

Comment:
Likewise, more info here....

`InvalidPower.New(fmt.Sprintf("%d", m.Power))`?

Which makes me think a `.Newf` method would be nice, so we could:
`InvalidPower.Newf("%d", m.Power)`

This would encourage more detailed messages.